.github/workflows/backend_publish.yml

```diff
@@ -34,13 +35,11 @@ jobs:
 
     - name: Build and Publish Docker Image
       working-directory: backend
-      env:
-        GITHUB_ACTOR: ${{ github.actor }}
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
         # Build the Docker image and tag it with both the 'latest', commit hash, and tag if exist
         ./gradlew bootBuildImage --imageName=ghcr.io/zabiny/klabis-be:latest
-        ./gradlew bootBuildImage --imageName=ghcr.io/zabiny/klabis-be:$(git rev-parse --short HEAD)
+        docker tag ghcr.io/zabiny/klabis-be:latest ghcr.io/zabiny/klabis-be:$(git rev-parse --short HEAD) 
+        docker push -a ghcr.io/zabiny/klabis-be 
```

Comment:
docker commands will not work without logging in first 
https://github.com/docker/login-action

Reply:
`docker` is [logged in|https://github.com/zabiny/klabis/pull/248/files/6e2fac83ca4778407ec2d99ff1a1642070d0019d#diff-1ada79d11a8d53da3cf6fafbab70a6a7d981c056dab14e2baa0b34b06bf11aeaL28] ... you wrote it ;) 

pipeline works - https://github.com/zabiny/klabis/actions/runs/10862180442/job/30144910105

Reply:
Pipeline jak je tady nastavena tak funguje - je otestovana napr. tady: https://github.com/zabiny/klabis/actions/runs/10862180442/job/30144910105  

Ostatne vypis vyse je prave z toho testu...

Reply:
> `docker` is logged in from previous step in pipeline ... you wrote that pipeline... ;)
> 
> ![image](https://private-user-images.githubusercontent.com/123363733/367539616-1ada97c6-4820-46a1-a387-b650ec884a42.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYzODYwMjMsIm5iZiI6MTcyNjM4NTcyMywicGF0aCI6Ii8xMjMzNjM3MzMvMzY3NTM5NjE2LTFhZGE5N2M2LTQ4MjAtNDZhMS1hMzg3LWI2NTBlYzg4NGE0Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxNVQwNzM1MjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03ZWVmNzJmOGU1OGYwZWNjZTFhNjUzY2Y4OWY5ZGMxOTU5ZWRkMDYyZDM1OTk0YmJhYjFiZjI0ZGEzYTlmZTQwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9._zqzf_eVnizje1yYh1ZfyFkLk-Q6cIPTA-8lCnTSHoE)

I got bamboozled by myself. It was not needed in my pipeline as the publish was done by the gradle task with credentials, so consequently I did not see it the _diff_ so I thought there is missing authorization. :clown_face: